changelog/_unreleased/2025-03-29-remove-debit-payment-method.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
as we will get rid of those single changelog files, would you mind adding this directly to the upgrade guide files ?

Reply:
@mitelg is this documented somewhere yet? It's completely new to me, that we are going to do this

Reply:
is not communicated yet. I am in that loop, so I thought it would be a good idea to already start, but we should wait until it is official :see_no_evil: sorry for bothering!